src/dswx_sar/initial_threshold.py

```diff
@@ -271,7 +273,7 @@ def tile_selection_wbd(self,
                            intensity,
                            water_mask,
                            win_size=200,
-                           selection_method='combined',
+                           selection_method=['combined'],
```

Comment:
What do you think about making this plural since more than one selection method is allowed?
```suggestion
                           selection_methods=['combined'],
```